tests/main/views/test_frameworks.py

```diff
@@ -378,6 +395,158 @@ def test_update_framework_catches_db_errors(self, db_commit, open_example_framew
             assert "Could not commit" in json.loads(response.get_data())["error"]
 
 
+class TestUpdateFrameworkPending(BaseApplicationTest):
+    EXPECTED_FRAMEWORK_DECLARATION_SUPPLIER_INFO = {
+        KEY_VAT_NUMBER: "123456789",
+        KEY_DUNS_NUMBER: "987654321",
+        KEY_TRADING_NAME: "Sam's Sweaters",
+        KEY_TRADING_STATUS: "limited company",
+        KEY_REGISTERED_NAME: "Sam's Sweaters",
+        KEY_ORGANISATION_SIZE: "small",
+        KEY_REGISTRATION_DATE: "2018-01-01T12:00:00",
+        KEY_REGISTRATION_COUNTRY: "gb",
+        KEY_REGISTRATION_BUILDING: "1 Sweater Lane",
+        KEY_REGISTRATION_TOWN: "Milton Keynes",
+        KEY_REGISTRATION_POSTCODE: "SW3 8T",
+        KEY_REGISTRATION_NUMBER: "000123000",
+    }
+
+    def setup(self):
+        super(TestUpdateFrameworkPending, self).setup()
+
+        self.framework = Framework(slug='g-cloud-10', name='G-Cloud 10', framework='g-cloud', status='open',
```

Comment:
IIRC: If you give this framework and `id` of over 100 it will be implicitly handled by the teardown in `BaseApplicationTest` and you can get rid of the teardown below.

Reply:
Magic! Seems to work. Will change.